### DISCUSSION TITLE
How to send a message to a specific channel

### DISCUSSION THREAD

**Discussion by dandanthedev** [discordjs/discord.js#9706] [Q&A] [answered]
Hello everyone, ive just installed discord.js but it has changed a lot since i last looked at it. I am trying to find out how to send a message to a specific channel but for some reason cant find it anywhere. How do i do this?

**Comment by sdanialraza** [comment, marked as answer]
Make sure you have the `Guilds` intent, and then you can do `<Client>.channels.cache.get("channel_id")`, and once you have the channel, you can use `.send()` on the channel
